Cargo.toml

```diff
@@ -320,6 +320,7 @@ ed25519-consensus = { version = "2.0.1", features = ["serde"] }
 either = "1.8.0"
 petgraph = "0.5.1"
 enum_dispatch = "^0.3"
+ethers = "2.0.14"
```

Comment:
Do we need this as global dep or can it be placed inside the `eth-light-client`?

Reply:
i think its necessary if we want to work with the convention *.workspace = true